lib/random.py

```diff
@@ -0,0 +1,187 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Generate pseudo random numbers. Should not be used for security purposes."""
+
+from __go__.math.rand import Uint32, Seed
+from __go__.math import Pow as _pow
```

Comment:
I took this convention from the cpython random.py code. all Math functions there are imported with leading underscores ... so it would fit to the others once the remaining code is inserted...